test/UnitTests/MSTest.Core.Unit.Tests/Attributes/DataRowAttributeTests.cs

```diff
@@ -55,5 +59,27 @@ public void GetDataShouldReturnDataPassed()
 
             CollectionAssert.AreEqual(new object[] { "mercury" }, dataRow.GetData(null).FirstOrDefault());
         }
+
+        [TestMethod]
+        public void GetDisplayNameShouldReturnAppropriateName()
+        {
+            var dataRowAttribute = new DataRowAttribute(null);
+
+            this.dummyTestClass = new DummyTestClass();
+            this.testMethodInfo = this.dummyTestClass.GetType().GetTypeInfo().GetDeclaredMethod("DataRowTestMethod");
+
+            var data = new string[] { "First", "Second", null };
+            var data1 = new string[] { null, "First", "Second" };
+            var data2 = new string[] { "First", null, "Second" };
+
+            var displayName = dataRowAttribute.GetDisplayName(this.testMethodInfo, data);
+            Assert.AreEqual("DataRowTestMethod (First,Second,)", displayName);
+
+            displayName = dataRowAttribute.GetDisplayName(this.testMethodInfo, data1);
+            Assert.AreEqual("DataRowTestMethod (,First,Second)", displayName);
+
+            displayName = dataRowAttribute.GetDisplayName(this.testMethodInfo, data2);
+            Assert.AreEqual("DataRowTestMethod (First,,Second)", displayName);
```

Comment:
This does look a bit odd, It does not immediately occur to me that the second parameter is a null.

Reply:
It looks like this is .Net default though and consumers are already used to this. So that should be ok - it still looks slightly odd though.

Reply:
Yes, looks a bit odd to me as well. But since we want a consistent behavior irrespective of which argument is null and hence this change.